libs/ts/firebase/adapter/src/lib/firebase-sdk-adapter.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    Auth,
+    createUserWithEmailAndPassword,
+    sendPasswordResetEmail,
+    signInWithEmailAndPassword,
+    signOut,
+    user,
+} from '@angular/fire/auth';
+import { FirebaseServiceFactoryUtility } from './utilities/firebase-service-factory.utility';
+import { Functions, httpsCallable } from '@angular/fire/functions';
+import { inject, InjectionToken } from '@angular/core';
+
+export const fireAuth = (auth: Auth) =>
+    FirebaseServiceFactoryUtility.create(auth, {
+        createUserWithEmailAndPassword,
+        sendPasswordResetEmail,
+        signInWithEmailAndPassword,
+        signOut,
+        user,
+    });
+
+export const FIRE_AUTH = new InjectionToken<ReturnType<typeof fireAuth>>(
+    'FIRE_AUTH'
+);
+
+export const provideFireAuth = () => ({
+    provide: FIRE_AUTH,
+    useFactory: () => fireAuth(inject(Auth)),
+});
```

Comment:
This is an effective and small way adapt the Firebase SDK methods to Angular. Could propose to @angular/fire or as its own NPM package.